test/unit/kubernetes-deploy/kubeclient_builder_test.rb

```diff
@@ -28,13 +28,14 @@ def test_build_client_from_multiple_config_files
     Kubeclient::Client.any_instance.stubs(:discover)
     # Set KUBECONFIG to include multiple config files
     dummy_config = File.join(__dir__, '../../fixtures/kube-config/dummy_config.yml')
-    kubeconfig = "#{ENV['KUBECONFIG']}:#{dummy_config}"
```

Comment:
This wasn't providing great multi-config coverage in our CI, which doesn't set the env var.